bin/crew

```diff
@@ -655,18 +655,25 @@ def update
       puts "Package file for #{package[:name]} not found. :(".lightred if @opt_verbose
       next
     end
-    if package[:binary_sha256] && package[:binary_sha256] != @pkg.get_binary_sha256(@device[:architecture])
-      canBeUpdated += 1
-      puts "#{@pkg.name} could be updated (rebuild)"
-    end
-    if package[:version] != @pkg.version
-      canBeUpdated += 1
+    differentVersion = package[:version] != @pkg.version ? true : false
```

Comment:
Do not use `if` with boolean results, the condition is already a boolean:
```ruby
differentVersion = (package[:version] != @pkg.version)
hasSha = @pkg.get_binary_sha256(@device[:architecture]).nil? || package[:binary_sha256].nil?
differentSha = (hasSha && package[:binary_sha256] != @pkg.get_binary_sha256(@device[:architecture]))
```

Reply:
This version of `differentSha` is triggering rubocop now...